### ISSUE TITLE
Wrong ID's generated by documentserver-generate-allfonts.sh

### ISSUE THREAD

**Issue by casKd-dev** [ONLYOFFICE/DocumentServer#512]
**Do you want to request a *feature* or report a *bug*?**
bug
**What is the current behavior?**
Font ID's have a 0 before them and seem to match the amount of numbers in the name as highest ID

Example:
```
0002
0227
1202
```
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

1. Put fonts into `/usr/share/fonts`
2. Run `/usr/bin/documentserver-generate-allfonts.sh`
3. Load a document on the online editor (Nextcloud)

**What is the expected behavior?**
Fonts should generate without a 0 before them

**Did this work in previous versions of DocumentServer?**
Did not test

**DocumentServer version:**
5.2.8.24 (Latest docker image)

**Operating System:**
Ubuntu 16.04 xenial

**Browser version:**
Vivaldi 2.3.1440.48 (Stable channel) (64-bit)
Revision 38c6381358a1ec7a9fc161a038fb9d7407334aa4-refs/branch-heads/3626@{#858}

**Comment by ShockwaveNN** [comment]
@casKd-dev Hi, I'm not sure that your problem exactly in 0 in font names, never seen problem like this.
Are you sure any other browsers are not cause any errors and please check that there is no errors in DocumentServer logs (`/var/log/onlyoffice`)

**Comment by casKd-dev** [comment]
There is nothing in the logs, it only happens when i try to add custom fonts, tried using other browser and clear the website cache but nothing changes.

**Comment by ShockwaveNN** [comment]
@casKd-dev that is the count of fonts do you have on server? more that 1000?

**Comment by casKd-dev** [comment]
> @casKd-dev what is the count of fonts do you have on server? more than 1000?

I have tried to install about 2100

**Comment by ShockwaveNN** [comment]
@casKd-dev oh, this may be a reason, I've never tried this count of fonts :)
Not sure I will be able to find so much fonts, so if you share archive with me (via any cloud storage and please share link to shockwavenn@gmail.com) and I'll try to reproduce your issue.

**Comment by ShockwaveNN** [comment]
Hi, I recieved your fonts and installed them - and I cant see a problem 
font with id 0002 is loaded for me
Maybe it's somehow related with (maybe) your custom nginx config?
![image](https://user-images.githubusercontent.com/668524/54116965-e7bb2080-4400-11e9-93f5-3f57d5bcacc4.png)

**Comment by casKd-dev** [comment]
> 3. Load a document on the online editor (Nextcloud)
Has this been tested with nextcloud? I have a feeling nextcloud is not returning the proper url's.

**Comment by ShockwaveNN** [comment]
> Has this been tested with nextcloud?

nope, I just checked on clean DocumentServer, I transfer this issue to nextcloud repo if so

**Comment by casKd-dev** [comment]
> I transfer this issue to nextcloud repo if so

Please do, it seems to be the doings of nextcloud here. Updated the issue to match needed info for that.
